noggin/templates/user.html

```diff
@@ -59,6 +59,12 @@ <h5 class="mb-2" id="user_username">{{ user.username }}</h5>
                   </div>
                 </li>
               {% endif %}
+              {% if user.website_url %}
+                <li class="list-group-item d-flex justify-content-between align-items-center">
+                  <strong title='{{_("RHBZ")}}'><i class="fa fa-fw fa-link" aria-hidden="true"></i> {{_("Website")}}</strong> 
+                  {{ user.website_url }}
```

Comment:
What about making a pop-out link here?

Reply:
done! yeah i forgot to make this a link!